llm_guard/input_scanners/anonymize_helpers/analyzer.py

```diff
@@ -57,15 +97,23 @@ def _get_nlp_engine() -> NlpEngine:
     if not spacy.util.is_package("en_core_web_sm"):
         spacy.cli.download("en_core_web_sm")
 
+    if not spacy.util.is_package("zh_core_web_sm"):
```

Comment:
Should we only download it when `ZH` is chosen?

Reply:
Yes, let me see how to do it.